drools-wb-webapp/src/main/resources/org/drools/workbench/DroolsWorkbench.gwt.xml

```diff
@@ -3,6 +3,8 @@
     "http://www.gwtproject.org/doctype/2.7.0/gwt-module.dtd">
 <module>
 
+  <inherits name="com.google.gwt.logging.Logging"/>
```

Comment:
Oee.. I cannot remember exactly, but log4J is available client side. @mbarkley @csadilek Would you please be so kind as to remind me (and Toni) of the prerequisites for use?